src/test/ui/resolve/issue-100365.stderr

```diff
@@ -0,0 +1,19 @@
+error[E0423]: expected value, found trait `Into`
+  --> $DIR/issue-100365.rs:2:16
+   |
+LL |     let addr = Into::<std::net::IpAddr>.into([127, 0, 0, 1]);
+   |                ^^^^^^^^^^^^^^^^^^^^^^^^-----
+   |                |
+   |                help: use the path separator to refer to an item: `Into::into`
```

Comment:
Hmm, I guess I should suggest the arguments, too.

Reply:
`span_to_snippet` seems fine, just be careful about what happens when you have a trait/type coming from a macro expansion.

also, can you bump this up to  a verbose suggestion? The way the `help:` currently renders is kinda confusing, imo, due to the overlapping spans.

Reply:
Okay, actually I now just use `Span::between` with a suggestion of `"::"`. No need for `span_for_snippet` (thank god!).
I quite like it, however the suggestion span is quite short (only one character, namely `.`).

I experimented with `span_suggestion_verbose` as you suggested (even though I use a different span now) and
in my opinion it's too verbose but please give me feedback on this.
It would look like this:

```
error[E0423]: expected value, found trait `Into`
 --> q.rs:6:16
  |
6 |     let addr = Into::<std::net::IpAddr>.into([127, 0, 0, 1]);
  |                ^^^^^^^^^^^^^^^^^^^^^^^^
  |
help: use the path separator to refer to an item
  |
6 |     let addr = Into::<std::net::IpAddr>::into([127, 0, 0, 1]);
  |                                        ~~

error[E0423]: expected value, found trait `Into`
  --> q.rs:10:15
   |
10 |     let xoo = Into::<'static, (), 4, i32>.into::<{1 + 2}, i32>(000);
   |               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
help: use the path separator to refer to an item
   |
10 |     let xoo = Into::<'static, (), 4, i32>::into::<{1 + 2}, i32>(000);
   |                                          ~~
```